WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -273,8 +273,9 @@ private void cancelUpload() {
     }
 
     private void dispatchUploadAction(@NonNull final MediaModel media) {
-        AppLog.i(T.MEDIA, "Dispatching upload action: " + media.getTitle());
-        media.setUploadState(UploadState.UPLOADING.toString());
+        AppLog.i(T.MEDIA, "Dispatching upload action for media with id: " + media.getId() +
```

Comment:
Nitpick but can we explicitly write "local id" here (and below), like in https://github.com/wordpress-mobile/WordPress-Android/blob/568616999496b328eacc29c921b64aa2aa97efad/WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java#L155-L155